### PR TITLE
add SKOS-Editor

### PR DESCRIPTION
**Pull request by NTsch**
This adds an application for modifying SKOS files via Mom, as requested by colleagues working on the IllUrk project. I've written a Wiki entry for it here: https://github.com/icaruseu/mom-ca/wiki/SKOS-Editor. Besides the _skos-editor_-app itself, it mostly affects the _index_- and _publication_-apps.

By far my most extensive addition to Mom so far, I've tried to stay close to the way things are currently done. Any tips, suggestions and corrections are much appreciated.

### CODE REVIEW

File: my/XRX/src/mom/app/skos-editor/service/edit-definition.service.xml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+            <xrx:expression>$skoscontent/skos:Concept[@rdf:about=$concept]/skos:definition</xrx:expression>
+        </xrx:variable>
+        <xrx:variable>
+            <xrx:name>$session-username</xrx:name>
+            <xrx:expression>xmldb:get-current-user()</xrx:expression>
+        </xrx:variable>
+    </xrx:variables>
+    <xrx:body>
+        {
+        let $log1 := util:log('error', concat('username: ', $session-username))
```
Comment (StephanMa) [inline my/XRX/src/mom/app/skos-editor/service/edit-definition.service.xml:55]: Totally fine for debuting purposes... but not for live. Could you please comment them in?

File: my/XRX/src/mom/app/skos-editor/service/my-collection-copy-skos.service.xml [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+        </xrx:variable>
+    </xrx:variables>
+    <xrx:init>
+        <xrx:processor>
+            <xrx:translateflag>false</xrx:translateflag>
+        </xrx:processor>
+    </xrx:init>
+    <xrx:body>
+        {
+        let $log1 := util:log('error', concat('$userdir: ', $userdir))
```
Comment (StephanMa) [inline my/XRX/src/mom/app/skos-editor/service/my-collection-copy-skos.service.xml:56]: Totally fine for debuting purposes... but not for live. Could you please comment them in?

File: my/XRX/src/mom/app/skos-editor/skos-edit.xqm [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+module namespace skos-edit = "http://www.monasterium.net/NS/skos-edit";
+
+import module namespace conf = "http://www.monasterium.net/NS/conf"
+at "../xrx/conf.xqm";
+
+declare namespace xmldb = "http://exist-db.org/xquery/xmldb";
+
+
+declare function skos-edit:object-uri-tokens($atomid as xs:string, $atom-tag-name as xs:string) as xs:string* {
```
Comment (StephanMa) [inline my/XRX/src/mom/app/skos-editor/skos-edit.xqm:33]: could you please add initial command with the purposes of each function?

**Comment by NTsch** [comment]
I've commented out the logs and added comments for the functions.

**Review by yngwi — APPROVED** [review]

**Review by StephanMa — APPROVED** [review]

**Pull request merged**